lib/optimizely/helpers/config_manager_utils.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+#
+#    Copyright 2019, Optimizely and contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+require 'optimizely/logger'
+require 'optimizely/config/datafile_project_config'
+module Optimizely
+  module Helpers
+    module ConfigManagerUtils
+      module_function
+
+      def create_project_config_from_datafile(datafile, logger, error_handler, skip_json_validation)
```

Comment:
THis doesn't need to be a util method. It can be a static method on the DatafileProjectConfig since this is tightly coupled with that class.